### PR TITLE
Create 2023-12-13-gift-guide.md

### PR DESCRIPTION
**Pull request by laneymangan**

### CODE REVIEW

File: _posts/2023-12-13-gift-guide.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+---
+layout: blog_post
+title: "Compiler's 2023 Holiday Gift Guide"
```
Comment (machikoyasuda) [inline _posts/2023-12-13-gift-guide.md:3]:
```suggestion
title: "Compiler’s 2023 Holiday Gift Guide"
```

File: _posts/2023-12-13-gift-guide.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+subtitle: "Searching for that perfect gift for the public service enthusiast in your life? Compiler's got you covered! Here is this year's round-up of our favorite transit-related gift ideas.
```
Comment (machikoyasuda) [inline _posts/2023-12-13-gift-guide.md:4]:
```suggestion
subtitle: "Searching for that perfect gift for the public service enthusiast in your life? Compiler’s got you covered! Here is this year's round-up of our favorite transit-related gift ideas.
```

File: _posts/2023-12-13-gift-guide.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+*(P.S. If you didn't make the holiday cut off this year, we think these make great gifts year-round!)*"
```
Comment (machikoyasuda) [inline _posts/2023-12-13-gift-guide.md:5]:
```suggestion
*(P.S. If you didn’t make the holiday cut off this year, we think these make great gifts year-round!)*"
```

File: _posts/2023-12-13-gift-guide.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+description: ""
+author:
```
Comment (machikoyasuda) [inline _posts/2023-12-13-gift-guide.md:7]: If this was a collaboration, you could write `Compiler staff` for author. If not, put author name here.

File: _posts/2023-12-13-gift-guide.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+excerpt: "Learn how to write a blog post, include images, blockquotes and more."
```
Comment (machikoyasuda) [inline _posts/2023-12-13-gift-guide.md:8]: This is what shows up here 
image

File: _posts/2023-12-13-gift-guide.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+I suggest you wear one of these token rings and every time you fist bump someone make a loud beeping noise to let them know they can now board the train. No? Ok fine, but this is still an awesome ring that from a distance looks like a beautiful, vintage band but up close you realize is an absolute relic.
+
+## [8. NYC Real-Time Subway Clock](https://www.etsy.com/listing/1290166234/nyc-realtime-subway-clock?click_key=b6b30f8f57cd4ba10f3bbc3ed29088e5aed6b156%3A1290166234&click_sum=98c95a73&ref=user_profile&frs=1&sts=1)
+Idea: hang this in your living room and set it to all your friends' preferred lines to make sure your holiday party lasts until the last possible minute. Idea: hang this in your bedroom to ensure you get as many possible zzz’s before rolling out your door and down the subway stairs. Idea: hang this in your kitchen to give you peace of mind that you can enjoy your coffee without missing your train, or more realistically, so you can throw back your burning hot espresso when you look up and realize the L is 1 minute away. Idea: buy this.
+!(white mug)[/assets/blog/white-glossy-mug-white-11oz-handle-on-right.jpg]
```
Comment (machikoyasuda) [inline _posts/2023-12-13-gift-guide.md:35]:
```suggestion
<figure>
    <img src="assets/blog/white-glossy-mug-white-11oz-handle-on-right.jpg" alt="An image of a white mug with an illustration of a Los Angeles Metro light rail map." />
    <figcaption>An image of a white mug with an illustration of a Los Angeles Metro light rail map. Photograph courtesy of Metro.</figcaption>
</figure>
```

File: _posts/2023-12-13-gift-guide.md (hunk 1 above)
Comment (machikoyasuda) [reply _posts/2023-12-13-gift-guide.md:3]: https://smartquotesforsmartpeople.com/

Comment (laneymangan) [reply _posts/2023-12-13-gift-guide.md:3]: I had no idea this was a thing!

Comment (machikoyasuda) [reply _posts/2023-12-13-gift-guide.md:3]: To make these suggestion changes, you can click **Commit suggestion** and then **Commit changes**

image

File: _posts/2023-12-13-gift-guide.md (hunk 3 above)
Comment (laneymangan) [reply _posts/2023-12-13-gift-guide.md]: @machikoyasuda I was hoping to make the P.S. portion of this italicized and possibly in a new paragraph. Is there a way to do that? I tried the *'s and put it on a new line but doesn't seem to be working (CC @angela-tran)

Comment (machikoyasuda) [reply _posts/2023-12-13-gift-guide.md]:
```suggestion
<br><br><p><em>P.S. If you didn’t make the holiday cut off this year, we think these make great gifts year-round!</em></p>
```

Comment (machikoyasuda) [reply _posts/2023-12-13-gift-guide.md]: Try this ☝️

**Comment by machikoyasuda** [comment]
Once this post goes live, we will also have to push up a small PR to add the Blog link to the Menu here:
image

File: _posts/2023-12-13-gift-guide.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
+---
+layout: blog_post
+title: "Compiler’s 2023 Holiday Gift Guide"
+subtitle: "Searching for that perfect gift for the public service enthusiast in your life? Compiler’s got you covered! Here is this year’s round-up of our favorite transit-related gift ideas.
```
Comment (angela-tran) [inline _posts/2023-12-13-gift-guide.md:4]: Copying from your message in Slack:

```suggestion
subtitle: "Searching for that perfect gift for the public service enthusiast in your life? Compiler’s got you covered! Here is this year’s round-up of our favorite transit-related gift ideas.
description: "Searching for that perfect gift for the public service enthusiast in your life? Compiler’s got you covered!"
```

**Comment by AnthonyRollins** [comment]
@laneymangan, this post is so good and funny; great job! @angela-tran @machikoyasuda thank you so much for all your help with this :)

File: _posts/2023-12-13-gift-guide.md (hunk 7 above)
Comment (angela-tran) [reply _posts/2023-12-13-gift-guide.md:4]: Oops, that was not the right place for me to insert that line... I fixed it in [539bcdf](https://github.com/compilerla/compiler.la/pull/150/commits/539bcdfea22d18fc60c67e82c0cdbed0e7229f7e) 😅

**Review by angela-tran — APPROVED** [review]
🚀

**Comment by angela-tran** [comment]
Merged in for @laneymangan since she does not have Write permissions on the repo

**Pull request merged**